packages/tools/devtools/devtools-example/package.json

```diff
@@ -72,14 +72,23 @@
 		"@fluidframework/build-common": "^2.0.0",
 		"@fluidframework/build-tools": "^0.22.0",
 		"@fluidframework/eslint-config-fluid": "^2.0.0",
+		"@fluidframework/test-utils": "workspace:~",
 		"@microsoft/api-extractor": "^7.34.4",
 		"@testing-library/dom": "^8.2.0",
 		"@testing-library/jest-dom": "^5.16.5",
 		"@testing-library/react": "^12.0.0",
 		"@testing-library/user-event": "^14.4.3",
+		"@types/chrome": "0.0.232",
+		"@types/expect-puppeteer": "2.2.1",
 		"@types/jest": "29.5.3",
+		"@types/jest-environment-puppeteer": "2.2.0",
+		"@types/jsdom": "^21.1.1",
+		"@types/jsdom-global": "^3.0.4",
```

Comment:
Do we need the jsdom dependencies? We're using the puppeteer environment now, so I don't think they're needed?

Reply:
Yes, we can remove the jsdom dependencies as well. Then should I also delete CounterWidget.test.tsx file as well?

Reply:
Cleaned up